### PR TITLE
doc: Revert the AT host sample name

### PR DESCRIPTION
**Pull request by umapraseeda**
ref:https://github.com/nrfconnect/sdk-nrf/pull/4897

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

File: docs/devices/FlashingCertificate/CLI.rst [hunk 1]
```diff
@@ -29,13 +29,13 @@ Provisioning of the certificate using CLI also results in the following actions:
 
 .. only:: v1_5_x
 
-    1. Programming of the `AT host sample <https://github.com/NordicSemiconductor/at_host-hex>`_ onto the device.
+    1. Programming of the `AT Client sample <https://github.com/NordicSemiconductor/at_host-hex>`_ onto the device.
```
Comment (peknis) [inline docs/devices/FlashingCertificate/CLI.rst]: The readme.md in the link target still says AT Host, which needs to be changed as well.

File: docs/devices/FlashingCertificate/CLI.rst [hunk 2]
```diff
@@ -29,13 +29,13 @@ Provisioning of the certificate using CLI also results in the following actions:
...
     #. Provisioning of the :ref:`created device credentials <aws-device-credentials>`.
     #. Programming of the :ref:`configured and built firmware <aws-firmware-configuration>` to the device.
 
 .. only:: not v1_5_x
 
-    * Programming of the `AT host sample <https://github.com/NordicSemiconductor/at_host-hex>`_ onto the device.
+    * Programming of the `AT Client sample <https://github.com/NordicSemiconductor/at_host-hex>`_ onto the device.
```
Comment (peknis) [inline docs/devices/FlashingCertificate/CLI.rst]: Same as above.

**Review by peknis — CHANGES_REQUESTED** [review]

File: docs/devices/FlashingCertificate/CLI.rst (hunk 1 above)
Comment (coderbyheart) [reply docs/devices/FlashingCertificate/CLI.rst]: I disagree. "AT Client" is the wrong name for the sample (however it is kept for historical reasons in NCS). The sample runs an AT command host, not a client, therefore the linked project has the correct name.

If link must match the title:

```suggestion
    1. Programming of the `AT Client sample <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nrf9160/at_client/README.html>`_ onto the device.
```

**Review by coderbyheart — APPROVED** [review]

File: docs/devices/FlashingCertificate/CLI.rst (hunk 1 above)
Comment (umapraseeda) [reply docs/devices/FlashingCertificate/CLI.rst]: Changed the link to point to the NCS sample.

**Review by peknis — APPROVED** [review]

**Pull request merged**